easybuild/easyconfigs/p/Python/Python-3.8.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,431 @@
+name = 'Python'
+version = '3.8.4'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.%(namelower)s.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['32c4d9817ef11793da4d0d95b3191c4db81d2e45544614e8449255ca9ae3cc18']
+
+dependencies = [
+    ('binutils', '2.35'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.33.0'),
+    ('XZ', '5.2.5'),
+    ('GMP', '6.2.0'),  # required for pycrypto
+    ('libffi', '3.3'),  # required for cryptography
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # Python 3.7 requires OpenSSL >= 1.0.2, uncomment line below if OS version is too old (e.g. CentOS 6)
+    # ('OpenSSL', '1.1.1h'),
+]
+
+osdependencies = [OS_PKG_OPENSSL_DEV]
+
+# symlink 'pip' command to 'pip3' that is included with Python installation
+# required so we can update pip to version included in extensions, using pip
+installopts = " && ln -s %(installdir)s/bin/pip3 %(installdir)s/bin/pip"
+
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+}
+
+
+# order is important!
+# package versions updated 13 October 2020
+exts_list = [
+    ('setuptools', '50.3.0', {
```

Comment:
Shouldn't this go *below* `pip`? How can we use `pip` to install `setuptools` if it's not there yet?

Reply:
somehow it still worked, so i kinda of just did that when diffing python2 and 3. I changed it.